code/supervised/Logistic_Regression/logistic_ex1.py

```diff
@@ -0,0 +1,52 @@
+from sklearn.linear_model import LogisticRegression
```

Comment:
FutureWarning: Default solver will be changed to 'lbfgs' in 0.22. Specify a solver to silence this warning. 
I got this warning from this code, I'm not sure what it means, was wondering if you knew?

Reply:
Yeah, the LogisticRegression objects can use multiple different 'solvers' (basically just methods of finding the solution), which would be specified in the object creation.  I'll add that specifier and the warning should go away.  I think we are running on different versions of sklearn.

Reply:
I never saw the warning, but it should be gone with the latest version.  Please verify this when you get a chance.  All that should have been necessary was the specifier in the object.

Reply:
Yeah the warning is gone, thanks the comments look good.